tests/console/openqa_review.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test openqa-review can be started (with runtime dependencies)
+# Maintainer: Oliver Kurz <okurz@suse.de>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
```

Comment:
no need for it, it's already on the right console when running after consoletests